src/main/java/graphql/GraphQLError.java

```diff
@@ -20,18 +22,19 @@
  * @see <a href="https://spec.graphql.org/October2021/#sec-Errors">GraphQL Spec - 7.1.2 Errors</a>
  */
 @PublicApi
+@NullMarked
 public interface GraphQLError extends Serializable {
 
     /**
      * @return a description of the error intended for the developer as a guide to understand and correct the error
      */
-    String getMessage();
+    @Nullable String getMessage();
```

Comment:
TODO for me - message in another ExceptionWhileDataFetching is not nullable, does this need to be consistent?